src/screens/InputUsername/index.js

```diff
@@ -266,7 +266,7 @@ const ChooseUsername = () => {
 
   return (
     <SafeAreaView style={styles.container}>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop has been added to the `StatusBar` component. This changes the appearance of the status bar content. Ensure this change is consistent across different screens and devices.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->